DrakeCollision::Element constructor uses this pointer in base member initializer list

MSVC complained about this one, and it's a valid complaint:
```
c:\cygwin64\home\twan\drake-distro\drake\systems\plants\collision\Element.h(21) : warning C4355: 'this' : used in base member initializer list
c:\cygwin64\home\twan\drake-distro\drake\systems\plants\collision\Element.h(25) : warning C4355: 'this' : used in base member initializer list
```

Relevant lines in Element:
```
      Element(const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity())
        : DrakeShapes::Element(T_element_to_local), id((ElementId) this) 
      {};

      Element(const DrakeShapes::Geometry& geometry, 
          const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity())
        : DrakeShapes::Element(geometry, T_element_to_local), id((ElementId) this) 
      {};
```
@avalenzu could you take a look if you have the time?

--- a/systems/plants/collision/Element.cpp
+++ b/systems/plants/collision/Element.cpp
@@ -6,6 +6,19 @@ using namespace std;
 
 namespace DrakeCollision
 { 
+  Element::Element(const Matrix4d& T_element_to_local)
+    : DrakeShapes::Element(T_element_to_local)
+  { 
+    id = (ElementId) this;
+  }
+
+  Element::Element(const DrakeShapes::Geometry& geometry, 
+      const Matrix4d& T_element_to_local)
+    : DrakeShapes::Element(geometry, T_element_to_local)
+  {
+    id = (ElementId) this;
+  }
+
   Element::Element(const Element& other) 
     : DrakeShapes::Element(other), id((ElementId) this)  
   {

--- a/systems/plants/collision/Element.h
+++ b/systems/plants/collision/Element.h
@@ -18,14 +18,10 @@ namespace DrakeCollision
 
   class DLLEXPORT_drakeCollision Element : public DrakeShapes::Element {
     public:
-      Element(const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity())
-        : DrakeShapes::Element(T_element_to_local), id((ElementId) this) 
-      {};
+      Element(const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity());
 
       Element(const DrakeShapes::Geometry& geometry, 
-          const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity())
-        : DrakeShapes::Element(geometry, T_element_to_local), id((ElementId) this) 
-      {};
+              const Eigen::Matrix4d& T_element_to_local = Eigen::Matrix4d::Identity());
 
       virtual ~Element(){};
 
@@ -38,10 +34,10 @@ namespace DrakeCollision
       virtual bool collidesWith( const Element* other) const { return true; };
 
     protected:
-
       Element(const Element& other); 
 
-      const ElementId id;
+    private:
+      ElementId id;
 
     public:
       EIGEN_MAKE_ALIGNED_OPERATOR_NEW

